### DISCUSSION TITLE
Using B: Variables

### DISCUSSION THREAD

**Discussion by WilliamRiker2701** [mpaperno/WASimCommander#16] [General]
Hi, is there some specific thing to do in order to manipulate B variables? I'm not being able to do it through the WASimUI.
They seem to always have value = 0

Thanks!

**Comment by mpaperno** [comment]
Hi,

It is not possible to access Bs and Os through a standalone WASM.  Apparently it is possible if you "inject" a custom WASM gauge into a model (add it into its model/panel config). But short of that, I haven't found any other way.

I've considered releasing a "gauge" which users could incorporate into specific aircraft if they wanted to, but I'm not sure how practical that would be, for the effort.  WASM gauges can still use SimConnect to communicate with the outside world, so that's not really an issue, but having to edit model configs just to get to a few uncontrollable buttons/knobs seems like a lot of bother.

Cheers,
-Max

**Reply by WilliamRiker2701** [reply]
Certanly looks like a lot of pain for little return. I hope Asobo does something about it...
